source/ReactLoadableSSRAddon.js

```diff
@@ -195,7 +195,9 @@ class ReactLoadableSSRAddon {
       return 0;
     };
 
-    return compilationChunks.reduce((chunks, chunk) => {
+    return (
+      WEBPACK_5 ? Array.from(compilationChunks) : compilationChunks
```

Comment:
As this _logic_ is being used on 2 places, it's a great idea to abstract it into an util, so we have code reusability.

E.g.

```js
// source/utils/toArray.js

/**
 * Ensure that given source is an array
 * @method toArray
 * @function
 * @param {*} source
 * @returns {*[]}
 */
export default function toArray(source) {
  return Array.isArray(source) ? source : [source];
}
```

And then we can finally use:

```suggestion
      toArray(compilationChunks)
```

Reply:
This is a `Set` to `Array` change, so `[source] doesn't work.  `

Reply:
Not a problem!